src/RoleCard/RoleCard.tsx

```diff
@@ -1,32 +1,29 @@
-import { Card, Image, Text, Button, Center, Anchor } from "@mantine/core";
+import { Card, Image, Text, Button, Center } from "@mantine/core";
 import "./RoleCard.css";
 
 export default function RoleCard({ role, link }) {
   return (
     // <Card className='roleCard' shadow="lg" radius="xl" p="xl" style={{ width: 'calc(33.333% - 20px)', marginBottom: '20px' }}>
     <Card className="roleCard" shadow="lg" radius="xl" p="xl">
```

Comment:
We should probably have a max size for the card since on larger screens they get a little over whelming
![Screenshot 2024-11-20 at 17 11 16](https://github.com/user-attachments/assets/07429355-d196-40c3-9892-31ac8ded9df6)

Reply:
I think it looks better now that I've changed it to space-evenly. Let me know if it's still an issue.